src/Fresnel/Fold.hs

```diff
@@ -17,10 +18,11 @@ import Data.Profunctor.Traversing
 import Data.Profunctor.Unsafe ((#.), (.#))
 import Fresnel.Bifunctor.Contravariant
 import Fresnel.Optic
+import Fresnel.Profunctor.Optical
 
 -- Folds
 
-type Fold s a = forall p . (Bicontravariant p, Cochoice p, Traversing p) => Optic' p s a
+type Fold s a = forall p . IsFold p => Optic' p s a
```

Comment:
Instead of listing all the specific constraints manually, they’ve been included as superclasses of the `IsFold` class (possibly transitively, via its relationships to `AffineFold` or `Traversal`.